README.md

```diff
@@ -69,7 +69,7 @@ type WithResponse<T> = WithResponseSuccess<T> | WithResponseFailure;
 Manta Style will generate either the error case or the normal case according to your declarations.
 Already have your list of error conditions and messages well-defined with your back end? You are in the luck! Manta Style will generate those cases for you just like in real world.
 
-This also gives us one more motivation to fine-tune the typing to our codebase.
```

Comment:
This is a verb and should use present form perhaps?

Reply:
oh i was being blind..sorry, will change it back.